web/src/components/tasks/RaffleBingoSteps.ts

```diff
@@ -0,0 +1,43 @@
+import { fetchTaskByFilename } from "@/lib/async-content-fetchers/fetchTaskByFilename";
+import { Task } from "@/lib/types/types";
+import { arrayOfTaskAgencies } from "@businessnjgovnavigator/shared/taskAgency";
+
+type RaffleBingoStepLabels = "Get an ID Number" | "From Your NJ Municipality";
+
+export const RaffleBingoSteps: {
+  stepLabel: RaffleBingoStepLabels;
+  fileName: string;
+  agencyId: string;
+}[] = [
+  {
+    stepLabel: "Get an ID Number",
+    fileName: "raffle-license-step-1",
+    agencyId: "games-of-chance-control-commission",
+  },
+  { stepLabel: "From Your NJ Municipality", fileName: "raffle-license-step-2", agencyId: "nj-municipality" },
+];
+
+export const getRaffleBingoTask = async (stepIndex: number): Promise<Task> => {
+  return await fetchTaskByFilename(RaffleBingoSteps[stepIndex].fileName);
+};
+
+export const getAgencyName = (agencyId: string): string => {
+  const issuingAgency = arrayOfTaskAgencies.find((agency) => agency.id === agencyId);
+  return issuingAgency!.name;
+};
+
+export const shouldDisplayContinueButton = (stepIndex: number): boolean => {
+  return stepIndex !== RaffleBingoSteps.length - 1;
+};
+
+export const shouldDisplayBackButton = (stepIndex: number): boolean => {
+  return stepIndex !== 0;
+};
+
+export const shouldDisplayMarkAsCompleteButton = (stepIndex: number): boolean => {
+  return stepIndex === RaffleBingoSteps.length - 1;
+};
```

Comment:
[dust] Tiny thing here but I find it a better practice to name things based on what values they represent instead of where they are used as it makes them more flexible and increases readability. One way you can directly see the benefit of this is that we could replace the two `shouldDisplayContinueButton` and `shouldDisplayMarkAsCompleteButton` functions with just one `isLastStep` function. In the other file where that function is used, you can tell exactly what that new name represents without having to jump into this file. I would go with `isNotFirstStep` or something similar for the other one.